src/views/registration/summary/title.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * This template renders the summary Title
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-block__tickets__registration__title">
+	<header>
+		<h2>
+			<a href="<?php the_permalink( $event_id ); ?>">
+				<?php echo get_the_title( $event_id ); ?>
```

Comment:
escape?

Reply:
I think `get_the_title` already has some functions in it that escapes the actual title. I can wrap it with an escape function, I didn't want to bloat that.